client/sidebar/sections/components/OmnichannelCallToggleReady.tsx

```diff
@@ -0,0 +1,71 @@
+import { Sidebar } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement, useEffect, useState } from 'react';
+
+import { useCallClient } from '../../../contexts/CallContext';
+import { useTranslation } from '../../../contexts/TranslationContext';
+
+export const OmnichannelCallToggleReady = (): ReactElement => {
+	const [agentEnabled, setAgentEnabled] = useState(false); // TODO: get from AgentInfo
+	const t = useTranslation();
+	const [registered, setRegistered] = useState(false);
+
+	const voipCallIcon = {
+		title: !registered ? t('Enable') : t('Disable'),
+		color: registered ? 'success' : undefined,
+		icon: registered ? 'phone' : 'phone-disabled',
+	};
+	const voipClient = useCallClient();
+
+	// TODO: move registration flow to context provider
```

Comment:
not possible yet?

Reply:
This can be done in a different PR afterwards